src/databricks/labs/ucx/install.py

```diff
@@ -120,7 +121,8 @@ def _run_configured(self):
 
     @staticmethod
     def run_for_config(
-        ws: WorkspaceClient, config: WorkspaceConfig, *, prefix="ucx", override_clusters: dict[str, str] | None = None
+            ws: WorkspaceClient, config: WorkspaceConfig, *, prefix="ucx",
+            override_clusters: dict[str, str] | None = None
```

Comment:
This change is irrelevant to this issue. Revert

Reply:
Done by the formatter